YOLayout/YOLayout_IMP.h

```diff
@@ -124,25 +124,27 @@ typedef enum {
 - (CGRect)setFrame:(CGRect)frame view:(id)view options:(YOLayoutOptions)options;
 
 /*!
- Set the (sub)view frame.
- If we are calculating sizeThatFits, this doesn't actually set the views frame.
- Use this value instead of view.frame since the views frame might not have been set.
+ Center view with size in frame.
  
- @param frame Frame
- @param inRect Rect to optionally position in for YOLayoutOptionsCenter, YOLayoutOptionsCenterVertical, YOLayoutOptionsRightAlign, etc.
+ @param size Desired size
+ @param frame In frame
  @param view View
- @param options Options for setFrame; See YOLayoutOptions for more info
  @result The view frame.
  */
-- (CGRect)setFrame:(CGRect)frame inRect:(CGRect)inRect view:(id)view options:(YOLayoutOptions)options;
+- (CGRect)centerWithSize:(CGSize)size frame:(CGRect)frame view:(id)view;
 
 /*!
  Set the (sub)view frame.
- Calls sizeThatFits on the view and then centers it. If inRect.height == 0, we only center horizontally.
- @param inRect The containing rectangle.
+ If we are calculating sizeThatFits, this doesn't actually set the views frame.
+ Use this value instead of view.frame since the views frame might not have been set.
+ 
+ @param size Size
+ @param inRect Rect to optionally position in for YOLayoutOptionsCenter, YOLayoutOptionsCenterVertical, YOLayoutOptionsRightAlign, etc.
```

Comment:
Minor: Don't want to suggest that this is optional since the origin is important even when not using YOLayoutOptionsCenter etc. Maybe:

    @param inRect Rect in which to position the view. `inRect.size` may be different than `size` when using this method with YOLayoutOptionsCenter, YOLayoutOptionsCenterVertical, YOLayoutOptionsRightAlign, etc.